resources/js/components/Kiosk.vue

```diff
@@ -495,9 +505,9 @@ export default {
                         this.axiosErrorToBugsnag(error)
                         this.$swal.fire({
                             title: 'Error',
-                            text: 'Unable to process data. Check your internet connection or try refreshing the page.',
+                            text: 'An unexpected error occurred. If this continues, post in SCC/RJ Slack #people-counter or contact developers@robojackets.org.',
```

Comment:
I don't think you need to mention the workspaces here (or email, no one is going to email us.)